encodings/BinaryEncoding.md

```diff
@@ -61,15 +61,17 @@ and use the default values.
 * `len` = 16
 
 Is the ID of the whole trace forest. It is represented as a 16-bytes array,
-e.g. (in hex), `4bf92f3577b34da6a3ce929d0e0e4736`. All bytes 0 is considered invalid.
+e.g. (in hex), `4bf92f3577b34da6a3ce929d000e4736`. All bytes 0 is considered invalid.  If converted
+to a 128 bit integer, these bytes are interpreted in big-endian order.
```

Comment:
I would not mention numbers and endianness anywhere and I would try to implement the library using byte[] if possible.

See data model also mention 8-byte array:
https://github.com/census-instrumentation/opencensus-proto/blob/master/trace/span.proto

I think it will be great to not have to deal with endianness. I know that the previous HTTP format used decimal for spanId and probably we will use big-endian there.

What do you think?

Reply:
If we were to mention endian-ness, we could do it in a FAQ or section at the bottom. mainly because it is common practice for better or for worse.

An example could say..

> We encourage IDs to be treated as opaque byte arrays or lower-hex strings. If you need to store them as numbers, be consistent with endian-ness. For example, if you store as big-endian, you'll need to retrieve them as big-endian as well.